openupgradelib/openupgrade_merge_records.py

```diff
@@ -758,8 +759,10 @@ def _change_generic(
         try:
             model = env[model_to_replace].with_context(active_test=False)
         except KeyError:
-            continue
-        if (model._table, res_id_column) in exclude_columns:
+            table = get_model2table(model_to_replace)
+        else:
+            table = model._table
```

Comment:
Don't do this in `else`, but next to the `model = env[model_to_replace].with_context(active_test=False)`. The same for all occurrences of this type.